arwen/cryptoapi/cryptoei.go

```diff
@@ -365,14 +368,67 @@ func v1_4_verifySecp256k1(
 		return 1
 	}
 
-	invalidSigErr := crypto.VerifySecp256k1(key, message, sig)
+	invalidSigErr := crypto.VerifySecp256k1(key, message, sig, uint8(hashType))
 	if invalidSigErr != nil {
 		return -1
 	}
 
 	return 0
 }
 
+//export v1_4_verifySecp256k1
+func v1_4_verifySecp256k1(
+	context unsafe.Pointer,
+	keyOffset int32,
+	keyLength int32,
+	messageOffset int32,
+	messageLength int32,
+	sigOffset int32,
+) int32 {
+	return v1_4_verifyCustomSecp256k1(
+		context,
+		keyOffset,
+		keyLength,
+		messageOffset,
+		messageLength,
+		sigOffset,
+		int32(secp256k1.ECDSADoubleSha256),
+	)
+}
+
+//export v1_4_encodeSecp256k1DerSignature
+func v1_4_encodeSecp256k1DerSignature(
```

Comment:
do the import for the new function on line 82 like the other imports.

Reply:
done